index.html

```diff
@@ -34,12 +34,11 @@ <h1>Tanja Süß</h1>
     <div class="tagfield">
```

Comment:
is there a `section` tag missing for the whole skills section like eg further down for the biography section!? maybe i didn´t see it :) if there is no one i would recommend one :)

Reply:
as you suggested i wrapped the whole skills with their tagfields in one section, thanks for that advice 😊